lib/screens/v2/candidate_generator/busway.ex

```diff
@@ -1,37 +1,39 @@
-defmodule Screens.V2.CandidateGenerator.Solari do
+defmodule Screens.V2.CandidateGenerator.Busway do
   @moduledoc false
 
   alias Screens.V2.CandidateGenerator
   alias Screens.V2.CandidateGenerator.Widgets
   alias Screens.V2.Template.Builder
   alias Screens.V2.WidgetInstance.{NormalHeader, Placeholder}
   alias ScreensConfig.Screen
+  alias ScreensConfig.V2.Busway
   alias ScreensConfig.V2.Header.CurrentStopName
-  alias ScreensConfig.V2.Solari
+
+  defmodule Deps do
+    @moduledoc false
+    defstruct now: &DateTime.utc_now/0,
+              departures_instances: &Widgets.Departures.departures_instances/1
+  end
```

Comment:
Ooh I like this. Much neater than adding a separate argument for each dependency. 👍